src/Modal/Modal.test.jsx

```diff
@@ -127,6 +127,25 @@ describe('<Modal />', () => {
       expect(modalFooter.find('button')).toHaveLength(1);
       expect(wrapper.find('button')).toHaveLength(1);
     });
+
+    it('renders with IE11-specific styling when IE11 is detected', () => {
+      const { MSInputMethodContext } = global;
+      const { documentMode } = global.document;
+
+      // mimic IE11
+      global.MSInputMethodContext = true;
+      global.document.documentMode = true;
+      wrapper = mount(<Modal {...defaultProps} />);
+      let modal = wrapper.find('.modal');
+      expect(modal.hasClass('is-ie11')).toEqual(true);
+
+      // return globals back to not IE11
+      global.MSInputMethodContext = MSInputMethodContext;
```

Comment:
What happens if `window.MSInputMethodContext` or `document.documentMode` is `false` in the `global` context? The following assertion would fail, right? Basically, this test is assuming that it is not executed with an IE11 browser? 

Would it be safer to assign either of these to `false` (and really the appropriate tests are when `MSInputMethodContext` is `false` and when `documentMode` is `false`) and assert that `is-ie11` doesn't exist? And then reset globals as part of an `afterEach`?